slick/src/main/scala/slick/jdbc/OracleProfile.scala

```diff
@@ -96,8 +96,7 @@ trait OracleProfile extends JdbcProfile {
   override def defaultTables(implicit ec: ExecutionContext): DBIO[Seq[MTable]] = {
     for {
       user <- SimpleJdbcAction(_.session.metaData.getUserName)
-      tables <- SQLActionBuilder(Seq("select TABLE_NAME from all_tables where OWNER = ?"), implicitly[SetParameter[String]].applied(user)).as[String]
-      mtables <- MTable.getTables(None, None, None, Some(Seq("TABLE"))).map(_.filter(t => tables contains t.name.name)) // FIXME: this should check schema and maybe more
+      mtables <- MTable.getTables(None, Some(user), None, Some(Seq("TABLE")))
```

Comment:
For Oracle, schemas and users are essentially the same thing and are in this context. http://dba.stackexchange.com/questions/37012/difference-between-database-vs-user-vs-schema